### ISSUE TITLE
Duplicated dev docs warning on docs theme

### ISSUE THREAD

**Issue by thibaudcolas** [django/djangoproject.com#2167]
On docs pages in the dev version like [Using Django](https://docs.djangoproject.com/en/dev/topics/), our "This document is for Django's development version," warning is duplicated in the page HTML. Once at the top of the `<body>`, and another time at the bottom.

This seems to come from this JS script, that creates a clone of the element and then adds it to the page:

https://github.com/django/djangoproject.com/blob/6fcf41e44aec3ec9d62fef126a8390b94a88224a/djangoproject/static/js/djangoproject.js#L162-L167

I don’t think this is a big issue but definitely not good for users of assistive tech that the same content is on the page multiple times. The JS should be simple to change, but I’m not sure how much the styling of the warning relies on there being two copies like this.

**Comment by adamzap** [comment]
The JavaScript has done it that way for a very long time, and I'd like to change it. See #1988.

I wonder if we'd be open to a new UI/UX for these warnings.

**Comment by SachinAditya** [comment]
Hi mentors,
I’d like to work on this issue. I’m currently reviewing the code to understand the cause of the duplicated warning and will propose a fix that resolves it while keeping accessibility and styling in mind.

I’ll share a pull request once it’s ready. Please let me know if there’s anything specific you’d like me to consider.

Thanks!

**Comment by adamzap** [comment]
@SachinAditya Thanks, but #2408 is already open and will close this issue.

**Comment by SaptakS** [comment]
This is fixed by #2408 . Closing.
